examples/typescript-backbone/js/app.ts

```diff
@@ -339,6 +396,20 @@ class AppView extends Backbone.View {
 		Todos.each(this.addOne);
 	}
 
+	// Filter out completed/remaining tasks
+	filter(criteria: string) {
+		taskFilter = criteria;
```

Comment:
I think it comes down to what you want to make global. TaskRouter is currently inside AppView and not exposed, plus AppView isn't exposed either. Only the TodoList is exposed via the global Todos and even the AppView listens to events through that. The data is needed in TodoView at the moment. So if you put the current filter in either TaskRouter or AppView you have to expose one or both of them and add a getter for TodoView to call.

I feel like I'm missing something about best practice in Backbone (with or without TypeScript). I usually like to leave the code in a better state than I found it but am not sure if I know enough to do that. If the TodoMVC app is this much of a mess I would not like to see how a "real" app came out!